src/main/java/org/isf/medical/dto/MedicalDTO.java

```diff
@@ -33,38 +30,41 @@ public class MedicalDTO {
 	@ApiModelProperty(notes="The id of the medical", example = "1", position = 1)
 	private Integer code;
 	
-	@NotEmpty(message="The product code is required")
+	//@NotEmpty(message="The product code is required")
```

Comment:
@ArnaudFonzam can we remove these messages or not (instead to just comment them)?

Reply:
it is better to remove them

Reply:
it is better to remove them